api/client/utils.go

```diff
@@ -260,7 +260,7 @@ func (cli *DockerCli) monitorTtySize(id string, isExec bool) error {
 	sigchan := make(chan os.Signal, 1)
 	gosignal.Notify(sigchan, signal.SIGWINCH)
 	go func() {
-		for _ = range sigchan {
+		for range sigchan {
```

Comment:
@vbatts confirmed to me, that we want to keep backwards compatibility for now. So @dkjer please revert this little change :)